packages/suite-desktop/src/support/DesktopTitlebar.tsx

```diff
@@ -1,20 +1,21 @@
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
+import { useSelector } from '@suite-hooks';
 import NoSSR from '@suite-support/NoSSR';
 import { colors, TrezorLogo, Icon } from '@trezor/components';
 import { isMac as isMacOS } from '@suite-utils/env';
 
 const TITLEBAR_HEIGHT = 28;
 const RESIZE_HANDLER_PADDING = 4;
 
-const Titlebar = styled.div`
+const Titlebar = styled.div<{ isTor: boolean }>`
     display: block;
     height: ${TITLEBAR_HEIGHT}px;
     width: 100%;
     position: fixed;
     z-index: 1000000;
     position: relative;
-    background: ${colors.NEUE_TYPE_DARK_GREY};
+    background: ${props => (props.isTor ? '#59316B' : colors.NEUE_TYPE_DARK_GREY)};
```

Comment:
we dont want to have this color in our pallete? it will be used only here?

Reply:
Only used here (for now) but still added to the colour palette as it's cleaner and it tells us what it is :slightly_smiling_face: 

6a0864d